.eslintrc.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  extends: 'standard',
+  ignorePatterns: [
```

Comment:
This is brilliant work @domoscargin 😊 

Another few of tweaks here as ESLint have fixed a bug we found

* https://github.com/eslint/eslint/pull/16550

So don't think we need `ignorePatterns` at all 🎉 

I'll explain what each of the workarounds previously did though:

1. The `'!.*'` workaround included dotfiles
2. The `'node_modules'` + `'node_modules/.*'` combo un-included package dotfiles
3. The `'package-lock.json'` ignore was me (sorry) planning to lint JSON but never added it

```suggestion
```

Reply:
Sorry should have checked this too

I couldn't see the `'src/javascripts/vendor/**/*'` directory, is it generated somewhere?

Reply:
> I couldn't see the 'src/javascripts/vendor/**/*' directory, is it generated somewhere?

Huh, that `'src/javascripts/vendor/**/*'` ignore has been part of the config for 5 years, but even [when it was introduced](https://github.com/alphagov/govuk-design-system/pull/174), it didn't exist. I guess Metalsmith might've built in place then copied or something?

Whatever the case, Metalsmith now only builds in `deploy/public/javascripts/vendor`, which is ignored anyway, so all good!

Reply:
> So don't think we need ignorePatterns at all 🎉

Cool! That's still in `govuk-frontend` atm, so I guess we can clean that up as well

Reply:
Yep, slowly but surely! Thanks 😊